Makefile

```diff
@@ -0,0 +1,80 @@
+# Default values
+csv ?= script/inputs/ynETH.csv
+json ?= script/inputs/ynETH.json
+network ?= mainnet
+deployerAccountName := $(shell grep '^DEPLOYER_ACCOUNT_NAME=' .env | cut -d '=' -f2)
+deployerAddress := $(shell grep '^DEPLOYER_ADDRESS=' .env | cut -d '=' -f2)
+
+# Default target
+.PHONY: all
+all: clean install build
+
+# Clean the cache and output directories
+.PHONY: clean
+clean:
+	rm -rf cache out
+
+# Install dependencies using forge
+.PHONY: install
+install:
+	forge install
+
+# Build the project using forge
+.PHONY: build
+build:
+	forge build
+
+# Compile the project using forge
+.PHONY: compile
+compile:
+	forge compile
+
+# Run the tests using forge
+.PHONY: test
+test:
+	forge test -vvv
+
+# Create a gas snapshot
+.PHONY: snapshot
+snapshot:
+	forge snapshot
+
+# Generate a coverage report
+.PHONY: coverage
+coverage:
+	forge coverage
+
+# Generate a coverage report and create an HTML report
+.PHONY: coverage-report
+coverage-report:
+	forge coverage --report lcov && genhtml lcov.info --output-directory coverage
+
+# Show the coverage report
+.PHONY: coverage-show
+coverage-show:
+	npx http-server ./coverage
+
+# Lint the Solidity files using forge fmt and solhint
+.PHONY: lint
+lint:
+	forge fmt --check && solhint "{script,src,test}/**/*.sol"
+
+# Format the Solidity files using forge fmt
+.PHONY: format
+format:
+	forge fmt --root .
+
+# make convert csv=script/input/ynETH.csv
+.PHONY: convert
+convert:
+	bash ./script/bash/convertCSVjson.sh ${csv}
+
+# make simulate json=script/inputs/ynETH.json network=mainnet
+.PHONY: simulate
```

Comment:
does not work:

```make simulate
forge script DeployEigenAirdrop --sig "run(string memory)" script/inputs/ynETH.json --rpc-url mainnet
[⠊] Compiling...
No files changed, compilation skipped
Script ran successfully.

== Logs ==
  Deployed EigenAirdrop implementation at address:  0x90193C961A926261B756D1E5bb255e67ff9498A1
  Deployed EigenAirdrop at address:  0xA8452Ec99ce0C64f20701dB7dD3abDb607c00496

## Setting up 1 EVM.

Failed with `revert: ERC1967: new implementation is not a contract`:

  [3568] → new <unknown>@0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496
    └─ ← [Revert] revert: ERC1967: new implementation is not a contract

Error:
Simulated execution failed.
make: *** [simulate] Error 1```

Reply:
@danoctavian I'm unable to figure out why its failing. Can only conclude that there is a bug with foundry itself. Earlier i was using create2 deployment and it was working fine. Even now i tried to include a {salt: SALT} to the deployment script and simulation ran fine without the (--account and --sender). 

For the time being @MrDeadCe11 has added the --account and --sender to the simulate script and it runs without any errors.

Reply:
let's not add it if there's no point to it.

This feature is not needed. All we need is:

1. Test
2. Deploy

Any extra things let's leave oout